_1327/templates/base.html

```diff
@@ -54,14 +54,10 @@
 							</li>
 						{% endif %}
 					{% endfor %}
-					<li class="dropdown pull-right{% if user.is_authenticated %} active{% endif %} login-button">
-						<a href="{% if user.is_authenticated %}{% url 'logout' %}{% else %}{% url 'login' %}{% endif %}" class="login-link"><span class="glyphicon glyphicon-user"></span></a>
-						<ul class="dropdown-menu">
-							{% if user.is_authenticated %}
-								<li class="disabled"><a href="#">{% trans "Logged in as" %} {{ user.username }}</a></li>
-							{% endif %}
-							{% if CAN_CREATE_INFORMATIONPAGE or CAN_CREATE_MINUTES or CAN_CREATE_POLL %}
-								<li role="separator" class="divider"></li>
+					{% if CAN_CREATE_INFORMATIONPAGE or CAN_CREATE_MINUTES or CAN_CREATE_POLL %}
```

Comment:
If a user is not allowed to create information pages, minutes or polls, but allowed to  edit menu items he can not see the menu. Is this intended?